worlds/faxanadu/__init__.py

```diff
@@ -160,19 +165,13 @@ def create_items(self) -> None:
                 for i in range(item.progression_count):
                     itempool.append(FaxanaduItem(item.name, ItemClassification.progression, item.id, self.player))
 
-        # Set up filler ratios
-        self.filler_ratios = {
-            item.name: item.count
-            for item in Items.items
-            if item.classification in [ItemClassification.filler, ItemClassification.trap]
-        }
-
+        # Adjust filler ratios
         # If red potions are locked in shops, remove the count from the ratio.
         self.filler_ratios["Red Potion"] -= red_potion_in_shop_count
 
         # Remove poisons if not desired
```

Comment:
Should change the comment

Reply:
I did update the comment. Not sure what exactly you mean

Reply:
Oh... I promise I can read XD